flzoo/cifar10/cifar10_fedpart_resnet18_config.py

```diff
@@ -0,0 +1,39 @@
+from easydict import EasyDict
+
+exp_args = dict(
+    data=dict(
+        dataset='cifar10',
+        data_path='./data/cifar10',
+        sample_method=dict(name='iid', train_num=500, test_num=100)
+    ),
+    learn=dict(
+        device='cuda:0',
+        local_eps=8,
+        global_eps=141,
+        batch_size=32,
+        optimizer=dict(name='adam', lr=1e-3),
+        test_place=['before_aggregation', 'after_aggregation'],
+        # Only fine-tune parameters whose name contain the keyword "fc".
+        finetune_parameters=dict(name='contain', keywords=['fc']),
+    ),
+    model=dict(
+        name='resnet18',
+        input_channel=3,
+        class_number=10,
+    ),
+    client=dict(name='base_client', client_num=40),
+    server=dict(name='base_server'),
+    group=dict(
+        name='base_group',
+        aggregation_method='avg',
+        include_non_param=False,
+    ),
+    other=dict(test_freq=1, logging_path='./logging/cifar10_fedpart_resnet18_iid')
+)
+
+exp_args = EasyDict(exp_args)
+
+if __name__ == '__main__':
+    from fling.pipeline import partial_model_pipeline
```

Comment:
Add some comments about how to run a corresponding centralized learning experiment. Maybe we can only change the pipeline? Add correspondin comments for all fedpart configs.